razorpay-payments.php

```diff
@@ -48,6 +48,8 @@ class WC_Razorpay extends WC_Payment_Gateway
 
         const DEFAULT_LABEL                  = 'Credit Card/Debit Card/NetBanking';
         const DEFAULT_DESCRIPTION            = 'Pay securely by Credit or Debit card or Internet Banking through Razorpay.';
+        const DEFAULT_SUCCESS_MESSAGE  = 'Thank you for shopping with us. Your account has been charged and your
```

Comment:
@ccrims0n  We cannot add the i8n method here.instead I have used in line number 205.